$ ./person_test.out --success

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
person_test.out is a Catch v2.8.0 host application.
Run with -? for options

-------------------------------------------------------------------------------
`get_fullname` test
-------------------------------------------------------------------------------
person/person_test.cpp:8
...............................................................................

person/person_test.cpp:10: PASSED:
  REQUIRE( person.get_fullname() == "Edsger Dijkstra" )
with expansion:
  "Edsger Dijkstra" == "Edsger Dijkstra"

-------------------------------------------------------------------------------
constructor empty firstname test
-------------------------------------------------------------------------------
person/person_test.cpp:13
...............................................................................

person/person_test.cpp:14: PASSED:
  REQUIRE_THROWS_AS( Person("", "Dijkstra"), std::invalid_argument )

===============================================================================
All tests passed (2 assertions in 2 test cases)

